src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Daniel Zerbino <notifications@github.com> writes:
> Each object has a parent, but GFF3 also allows a derived_from tag, kinda like a
> secondary ancestor link. That means that although Gene1 is parent to
> Transcript1 and Transcript1 is parent to Exon1, you are free to have Exon1
> derived from another gene.


Also, I am looking at the spec again and remembering some of the
other uses of multiple parents.  I need to draw some pictures,
but it actually feels like a very elegant solution to some ugly
problems.

That Lincoln is a smart and pragmatic guy...